### PR TITLE
Blog post announcing the user personas questionnaire. 

### PR DESCRIPTION
**Pull request by yns000**
Blog post announcing the user personas questionnaire.

### CODE REVIEW

File: site/content/blog/2023-12-18-user-personas-poll/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Discovering Our Users - The ZAP User Personas Questionnaire"
+summary: >
+  Improving usability requires the identification of the type of users ZAP has
+images:
+- https://www.zaproxy.org/blog/2023-12-18-user-personas-poll/535a4e71-d6a6-40a1-8289-f31e8810b65b.png
+type: post
+tags:
+- blog
+- update
```
Comment (psiinon) [inline site/content/blog/2023-12-18-user-personas-poll/index.md:10]: `questionnaire` would be better than `update`

File: site/content/blog/2023-12-18-user-personas-poll/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Discovering Our Users - The ZAP User Personas Questionnaire"
+summary: >
+  Improving usability requires the identification of the type of users ZAP has
+images:
+- https://www.zaproxy.org/blog/2023-12-18-user-personas-poll/535a4e71-d6a6-40a1-8289-f31e8810b65b.png
```
Comment (psiinon) [inline site/content/blog/2023-12-18-user-personas-poll/index.md:6]: Not the stanard social medi size .. but doe that matter?

File: site/content/blog/2023-12-18-user-personas-poll/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Discovering Our Users - The ZAP User Personas Questionnaire"
+summary: >
+  Improving usability requires the identification of the type of users ZAP has
```
Comment (psiinon) [inline site/content/blog/2023-12-18-user-personas-poll/index.md:4]: This appears on https://www.zaproxy.org/blog/ and doesnt read that well?

**Review by psiinon — COMMENTED** [review]
Looks good, just some minor comments

File: site/content/blog/2023-12-18-user-personas-poll/index.md (hunk 2 above)
Comment (thc202) [inline site/content/blog/2023-12-18-user-personas-poll/index.md:6]: We usually add the images under an `images` directory.

File: site/content/blog/2023-12-18-user-personas-poll/index.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+images:
+- https://www.zaproxy.org/blog/2023-12-18-user-personas-poll/535a4e71-d6a6-40a1-8289-f31e8810b65b.png
+type: post
+tags:
+- blog
+- update
+date: "2023-12-19"
```
Comment (thc202) [inline site/content/blog/2023-12-18-user-personas-poll/index.md:11]: Needs update.

**Comment by psiinon** [comment]
Also link to the Questionnaire from the front page - basically undoing this change: https://github.com/zaproxy/zaproxy-website/pull/2300/files (but we can also do that in another PR if you like)

File: site/content/blog/2023-12-18-user-personas-poll/index.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+authors:
+- yiannis
+---
+
+## Discovering Our Users - The ZAP User Personas Questionnaire
+
+The Zed Attack Proxy (ZAP) team is dedicated to making our tool the best it can be for our diverse user base. As the cybersecurity landscape evolves, so do the needs of our users. That's why we're excited to announce the **ZAP User Personas Questionnaire** — an initiative to better understand and cater to the unique requirements of everyone who uses ZAP.
+
+## **Why This Matters**
```
Comment (psiinon) [inline site/content/blog/2023-12-18-user-personas-poll/index.md:20]: Must admit I tend to use H3 ie `###` for blog post sub headings..

**Comment by psiinon** [comment]
<details><summary>Screenshot</summary>
<p>

Screenshot 2023-12-18 at 15-51-58 Discovering Our Users - The ZAP User Personas Questionnaire


</p>
</details>

File: site/content/blog/2023-12-18-user-personas-poll/index.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+type: post
+tags:
+- blog
+- update
+date: "2023-12-19"
+authors:
+- yiannis
+---
+
+## Discovering Our Users - The ZAP User Personas Questionnaire
```
Comment (thc202) [inline site/content/blog/2023-12-18-user-personas-poll/index.md:16]: This one is redundant, the blog template already includes the title above the content.

File: site/content/blog/2023-12-18-user-personas-poll/index.md (hunk 4 above)
Comment (thc202) [reply site/content/blog/2023-12-18-user-personas-poll/index.md:11]: No longer needs update, but the dir name does.

File: site/content/blog/2023-12-18-user-personas-poll/index.md (hunk 2 above)
Comment (kingthorin) [reply site/content/blog/2023-12-18-user-personas-poll/index.md:6]: I'm not 100% sure but I believe most sites will resize 🤷‍♂️

File: site/content/blog/2023-12-18-user-personas-poll/index.md (hunk 1 above)
Comment (kingthorin) [reply site/content/blog/2023-12-18-user-personas-poll/index.md:10]: Seems to have been missed somehow

File: site/content/blog/2023-12-18-user-personas-poll/index.md (hunk 5 above)
Comment (kingthorin) [reply site/content/blog/2023-12-18-user-personas-poll/index.md:20]: Still looks like two and bold to me

File: site/content/blog/2023-12-18-user-personas-poll/index.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+### **Goals of the Questionnaire**
+- **Identify Distinct User Personas:** We aim to pinpoint the various user types and their specific needs.
+- **Tailor Development to User Needs:** Your feedback will directly influence future updates and features.
+- **Enhance User Experience:** We're committed to making ZAP as user-friendly and effective as possible.
+
+### **Participate in the Questionnaire**
+We invite all ZAP users to participate in this questionnaire. Your input is invaluable in shaping the future of ZAP. The questionnaire includes questions about your role, your experience with ZAP, challenges you face, and features you wish to see.
+
+[🔗 **Start the ZAP User Personas Questionnaire**](https://github.com/zaproxy/zaproxy-website/pull/2300/files)
```
Comment (thc202) [inline site/content/blog/2023-12-18-user-personas-poll/index.md:29]: Wrong link?

File: site/content/blog/2023-12-18-user-personas-poll/index.md (hunk 4 above)
Comment (thc202) [reply site/content/blog/2023-12-18-user-personas-poll/index.md]: While not necessary the dir name/link could be changed as well.

**Comment by psiinon** [comment]
Arg sorry, tried to push a tweak but ended up pushing too much 😦 
Will fix ...

**Comment by psiinon** [comment]
Hopefully fixed now..

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you both!

**Review by psiinon — APPROVED** [review]

**Pull request merged**